lessons/module-1/js-classes.md

```diff
@@ -6,29 +6,33 @@ tags: js, introduction, constructor functions, this, classes, objects
 
 ## Learning Goals
 
-* Understand what `this` is and how it changes based on context
-* Understand what a class is and what object instances are
+- Understand what `this` is and how it changes based on context
+- Understand what a class is and what object instances are
 
-## Vocabulary
+## New Vocabulary
 
-- `Object` A bundle of behavior (methods) and state (properties)
-- `Key` The name used to reference a Value on an Object
-- `Value` The data referenced by a Key
 - `Property` Another word for a key-value pair on an object
 - `Method` A function on an Object
-- `Dot Notation` Notation to access a Value on an Object, explicitly specifies the Key
-- `Bracket Notation` Notation to access a Value on an Object, usually specifies a Key via a variable
 - `this` A JavaScript keyword with a value that changes depending on the context in which it's used
 - `Class` A constructor that allows us to create multiple instances
 - `Object Instance` Objects that contain the data and functionality defined in the class
 
+## Familiar or Save-for-Later Vocab
```

Comment:
Does the "save for later" include words that will be addressed later in the lesson once the fundamental new concepts are covered?

Reply:
I'm imagining it as vocab that will be covered more in later lessons. For example, bracket and dot notation has an entire lesson devoted to it later in Mod 1.